test/rest_api/test_storage_service.py

```diff
@@ -26,7 +26,6 @@ def test_storage_service_keyspaces(cql, this_dc, rest_api):
         resp_user.raise_for_status()
         keyspaces_user = resp_user.json()
         assert keyspace in keyspaces_user
-        assert all(not ks.startswith("system") for ks in keyspaces_user)
```

Comment:
I would have liked to leave an assert that checks that at least **something** isn't in keyspaces_user - otherwise how do we test that it returns only user keyspaces and not all of them. What about just checking that "system" itself (and not anything starting with system) isn't in keyspaces_user?

Reply:
Fair enough. Updated